src/ui/sso/components/attributeSelectionSection.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import { StateAttributeSummary } from 'src/reducers/sso'
+import { Block } from 'src/ui/structure'
+import { EmailIcon, PhoneIcon, NameIcon } from 'src/resources'
+import { View, Text, StyleSheet } from 'react-native'
+import { JolocomTheme } from 'src/styles/jolocom-theme'
+import { AttributeCard } from 'src/ui/sso/components/ssoAttributeCard'
+import { areCredTypesEqual } from 'src/lib/util'
+
+interface Props {
+  attributeType: string[]
+  attributes: StateAttributeSummary[]
+  handleAttributeSelection: Function
+}
+
+interface State {
+  selectedAttribute: string
+}
+
+// TODO horisontal flex container abstraction
+const styles = StyleSheet.create({
+  attributeSelectionContainer: {
+    flexDirection: 'row', 
+    justifyContent: 'space-between', 
+    alignItems: 'flex-start',
+    backgroundColor: JolocomTheme.primaryColorWhite,
+    paddingVertical: '5%',
+    marginBottom: '1%'
+  },
+  attributeTitle: {
+    ...JolocomTheme.textStyles.light.labelDisplayFieldEdit,
+    fontFamily: JolocomTheme.contentFontFamily
+  }
+})
+
+export class AttributeSummary extends React.Component<Props, State>{
+  state = {
+    selectedAttribute: ''
+  }
+
+  // TODO Screen for selecting specific verification
+  // TODO When would no verifications / attribute info be available?
+  private handleAttributeSelection = (attribute: string) => {
+    const attributeInfo = this.props.attributes.find(attr => attr.value === attribute)
+
+    if (!attributeInfo) {
+      return
+    }
+
+    const availableVerifications = attributeInfo.verifications
+    const { attributeType, handleAttributeSelection } = this.props
+
+    switch (availableVerifications.length){
+      case 0:
+        alert('No verifications found, weird')
+        return
+      case 1:
+        this.setState({selectedAttribute: attribute})
+        return handleAttributeSelection(attributeType, availableVerifications[0])
+      default:
+        alert('More than one verification found, sending the first one for now')
+        this.setState({selectedAttribute: attribute})
+        return handleAttributeSelection(attributeType, availableVerifications[0])
+    }
+  }
+
+  private renderAvailableAttributes = (attributes: StateAttributeSummary[]) => {
+    return attributes.map(attribute =>
+      <AttributeCard
+        key={attribute.value}
+        attributeValue={attribute.value}
+        onCheck={this.handleAttributeSelection}
+        checked={attribute.value === this.state.selectedAttribute}
+      />
+    )
+  }
+
+  // TODO Use common function instead, perhaps move to util
+  // TODO Better default value
+  private getNameAndIconByType(type: string[]) {
+    const map = [{
```

Comment:
We should open an issue to actually unify display helper mappings like this one between home screen and consent screen.